win32ss/user/user32/controls/listbox.c

```diff
@@ -321,9 +321,14 @@ static LRESULT LISTBOX_SetTopItem( LB_DESCR *descr, INT index, BOOL scroll )
         }
         else
             diff = (descr->top_item - index) * descr->item_height;
-
-        ScrollWindowEx( descr->self, 0, diff, NULL, NULL, 0, NULL,
-                        SW_INVALIDATE | SW_ERASE | SW_SCROLLCHILDREN );
```

Comment:
Since the purpose of the `#ifdef __REACTOS__` is to keep the Wine code intact, your diff generally should not contain any `-` (red) lines.
In this particular case you'll want to undo the indentation and formatting change.
Thanks.

Reply:
Done

Reply:
There are still changes at lines: 24, 324, 520, 1078, 2633, 2634, 3149

Reply:
Right, those are just trimming trailing whitespace, which I consider okay.
However for future reference, it is good practice to keep your diff minimal and avoid such changes.